tests/app/test_jinja_filters.py

```diff
@@ -90,6 +92,20 @@ def test_format_date_time_in_gmt(self):
             format_value, "<span class='date'>28 October 2018 at 11:59</span>"
         )
 
+    def test_format_date_using_london_timezone(self):
+        # Given a London timezone
+        london_timezone = timezone("Europe/London")
+        loc_dt = london_timezone.localize(datetime(2021, 8, 3, 00, 15, 0))
```

Comment:
Variable names shouldn't be shortened, it's also clear from reading the test that it will be a localised datetime, so this could just be named something like `date_time`.

Reply:
I lifted this from their docs :laughing: